### ISSUE TITLE
May 6, 2020

### ISSUE THREAD

**Issue by Joshua-Smith-42** [alan412/LearnJavaForFTC#2]
# Date on version you are commenting on
May 1, 2020

# Section you are commenting on
example - 3.2

# Comments
I was confused about xxx

**Comment by itsteja17** [comment]
May 7, 2020

4.6

No period at the end of the footnote. If you scroll down to other footnotes, they all have periods.
--
No space after “gampad1.a”

**Comment by alan412** [comment]
Good point.   I'll start with a clean set and see what it asks and how to explain what to do.
